packages/nextjs/components/Footer.tsx

```diff
@@ -12,21 +14,32 @@ import { getTargetNetwork } from "~~/utils/scaffold-eth";
  */
 export const Footer = () => {
   const nativeCurrencyPrice = useGlobalState(state => state.nativeCurrencyPrice);
+  const isLocalNetwork = getTargetNetwork().id === hardhat.id;
 
   return (
     <div className="min-h-0 py-5 px-1 mb-11 lg:mb-0">
       <div>
-        <div className="fixed flex justify-between items-center w-full z-10 p-4 bottom-0 left-0 pointer-events-none">
-          <div className="flex space-x-2 pointer-events-auto">
+        <div className="fixed flex justify-between items-center w-full z-10 p-4 bottom-0 left-0 pointer-events-none ">
+          <div className="flex flex-col md:flex-row gap-2 pointer-events-auto">
             {nativeCurrencyPrice > 0 && (
-              <div className="btn btn-primary btn-sm font-normal cursor-auto gap-0">
-                <CurrencyDollarIcon className="h-4 w-4 mr-0.5" />
-                <span>{nativeCurrencyPrice}</span>
+              <div>
+                <div className="btn btn-primary btn-sm font-normal cursor-auto gap-0">
+                  <CurrencyDollarIcon className="h-4 w-4 mr-0.5" />
+                  <span>{nativeCurrencyPrice}</span>
+                </div>
               </div>
             )}
-            {getTargetNetwork().id === hardhat.id && <Faucet />}
+            {isLocalNetwork && (
+              <>
+                <Faucet />
+                <Link href="/blockexplorer" passHref className="btn btn-primary btn-sm font-normal gap-0 normal-case">
+                  <MagnifyingGlassIcon className="h-4 w-4 mr-0.5" />
+                  <span>Block Explorer</span>
+                </Link>
```

Comment:
A few variations here that we could try too:

- Only the Glass Icon
- Only :mag: Explorer
- Switch places with the Faucet...

Reply:
- 🔍 ScaffoldEthScan  (or Se2-Scan) 😄

Reply:
I like it with the current icon + text 👌

Reply:
Yeah even I like current sequence and icon + text 🙌